code/modules/reagents/Chemistry-Colours.dm

```diff
@@ -8,17 +8,19 @@
 
 	var/color
 	var/reagent_color
+	var/i = 0

```

Comment:
probably iteration, since thats what ```i``` in a loop usually stands for, the iteration of the loop.